electron/channels/get-room-channel.js

```diff
@@ -1,12 +1,25 @@
 const { ipcMain } = require('electron')
 const { channels } = require('../../src/shared/constants')
 const { getRoom } = require('../lib/room-util')
+const UserDataStore = require('electron-store')
 
-ipcMain.on(channels.GET_ROOM, (event, path) => {
+ipcMain.on(channels.GET_ROOM, (event) => {
   console.log('GET_ROOM', path)
-  getRoom(path).then((room) => {
-    console.log(JSON.stringify(room, null, 2))
+  const store = new UserDataStore()
+
+  const defaultRoom = {
+    roomId: 'welcome-room',
+    path: './public/default-welcome-room',
+    dioryIdInFocus: 'welcome-room',
+  }
+
+  const roomInFocus = store.get('roomInFocus') || defaultRoom
```

Comment:
I would store just path in GET_PATH/SET_PATH or similar (see more below)

Reply:
This won't be done by now. Let's consider and change the architecture later...